src/client/index.ts

```diff
@@ -0,0 +1,19 @@
+'use client';
+import { default as ConfigProvider, ConfigProviderProps, useConfig } from './use-config';
+import {
+  default as UserProvider,
+  UserProviderProps,
+  UserProfile,
+  UserContext,
+  RequestError,
+  useUser
+} from './use-user';
+import {
+  default as withPageAuthRequired,
+  WithPageAuthRequired,
+  WithPageAuthRequiredProps,
+  WithPageAuthRequiredOptions
+} from './with-page-auth-required';
+export { ConfigProvider, ConfigProviderProps, useConfig };
```

Comment:
Do we want to export `useConfig` given that it's only used internally?

Reply:
Yeah, that's a fair point, will fix

Reply:
7c6e917